dumux-precice/couplingadapter.cc

```diff
@@ -58,10 +57,8 @@ void CouplingAdapter::setMesh(const std::string &meshName,
                               const std::vector<double> &positions)
 {
     assert(wasCreated_);
-    vertexIDs_ =
-        std::vector<int>(positions.size() / getMeshDimensions(meshName));
-    vertexIDsSpan_ = precice::span(vertexIDs_);
-    precice_->setMeshVertices(meshName, positions, vertexIDsSpan_);
+    vertexIDs_.resize(positions.size() / getMeshDimensions(meshName));
+    precice_->setMeshVertices(meshName, positions, vertexIDs_);
```

Comment:
Wasn't the idea of introducing `precice::span` that the user should also use it? Or not?